### DISCUSSION TITLE
How to detect HOMED status?

### DISCUSSION THREAD

**Discussion by stevempotter** [grblHAL/core#226] [Q&A] [answered]
ioSender can home the machine, or my pendant can home the machine. What do I need to put in my pendant code to let it know whether the machine was already homed by ioSender? I could not find any "homed" status items in the grbl status reports. 
I presume there is one "homed" status for each axis. How do I query that?

**Comment by terjeio** [comment, marked as answer]
Use `$#` - a [HOME element](https://github.com/grblHAL/core/wiki/Report-extensions#ngc-parameters-report-extensions) will be added when homing is enabled. It contains the homed position and an axismap telling which axes has been homed.

**Reply by stevempotter** [reply]
Great, I see:
$# 
...
[HOME:0.000,0.000,0.000:0]
...

"axismap": I think that means the last character (after the coordinates and the colon) will be a '7' if all three axes are homed, and a '0' if none are, correct?

**Reply by terjeio** [reply]
Correct - for a 3 axis machine.
